test/linter/test-obsolete.js

```diff
@@ -0,0 +1,110 @@
+/* This file is a part of @mdn/browser-compat-data
+ * See LICENSE file for more information. */
+
+import chalk from 'chalk-template';
+import bcd from '../../index.js';
+const { browsers } = bcd;
+
+const exceptions = [
+  'api.VRDisplay.hardwareUnitId',
+  'api.VREyeParameters.recommendedFieldOfView',
+  'api.VREyeParameters.renderRect',
+  'api.VRFieldOfView.VRFieldOfView',
+  'api.VRPose.hasOrientation',
+  'api.VRPose.hasPosition',
+  'css.types.length.lh',
+  'css.types.length.rlh',
+  'http.headers.Cache-Control.stale-if-error',
+  'http.headers.Feature-Policy.layout-animations',
+  'http.headers.Feature-Policy.legacy-image-formats',
+  'http.headers.Feature-Policy.oversized-images',
+  'http.headers.Feature-Policy.unoptimized-images',
+  'http.headers.Feature-Policy.unsized-media',
+  'svg.attributes.presentation.color-rendering',
+  'svg.elements.view.zoomAndPan',
+];
+
+function neverImplemented(support) {
+  for (const s in support) {
+    let data = support[s];
+    if (!Array.isArray(data)) data = [data];
+    for (const d of data) if (d.version_added) return false;
+  }
+  return true;
+}
+
+/**
+ * @param {*} browsers
+ * @param {*} support
+ * @returns 'warning' | 'error' | false
+ */
+function implementedAndRemoved(browsers, support) {
+  let result = 'error';
+  const warningTime = new Date().getTime() - 2 * 365 * 24 * 60 * 60 * 1000;
+  const errorTime = new Date().getTime() - 2.5 * 365 * 24 * 60 * 60 * 1000;
+  for (const browser in support) {
+    let data = support[browser];
+    if (!Array.isArray(data)) data = [data];
+    for (const d of data) {
+      // Feature is still supported
+      if (!d.version_removed) return false;
+      const releaseDate = new Date(
+        browsers[browser].releases[d.version_removed].release_date,
+      ).getTime();
```

Comment:
The `.getTime()` call can be removed since we can compare dates using comparison operators directly:

```suggestion
      );
```